### PR TITLE
Add new name "System" and new domain

### PR DESCRIPTION
**Pull request by maya**
**To be merged right after the domain name change**

This adds the new name, shorthand, project domain, GitHub URL, and email address.

Related: https://github.com/18F/web-design-standards-docs/pull/487.

See #2300

### CODE REVIEW

File: WHO_IS_USING_USWDS.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Website and applications that use the Standards
+# Website and applications that use the Design System
```
Comment (thisisdano) [inline WHO_IS_USING_USWDS.md]: `Websites`?

File: WHO_IS_USING_USWDS.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-Below are a list of website and applications currently using the U.S. Web Design Standards. If your project is currently using the Standards and you do not see it on this list, please feel free to submit a pull request or email the core team at uswebdesignstandards@gsa.gov.
+Below are a list of website and applications currently using the U.S. Web Design System. If your project is currently using the Design System and you do not see it on this list, please feel free to submit a pull request or email the core team at uswebdesignstandards@gsa.gov.
```
Comment (thisisdano) [inline WHO_IS_USING_USWDS.md]: `websites`?

**Comment by thisisdano** [comment]
Seeing all these together makes me wonder whether we should use capitals when referring to the design system in shorthand. As in 

`If your project is currently using the Design System and you do not see it on this list...`
vs
`If your project is currently using the design system and you do not see it on this list...`

I see how caps is consistent with our earlier usage, but I wonder if it's the right call now.

**Comment by maya** [comment]
@thisisdano caps should be used if Design System is a proper noun, but lowercase if it's more of a general thing.

**Comment by thisisdano** [comment]
Yeah, I get the proper noun thing, but that usage does have a whiff of incorrectness even though it's technically correct, like.... Internet or... Kleenex. But I'm comfortable moving forward with it now. I don't think this is a decision that can't be modified at any later date if it seems more wrong/weird.

**Comment by maya** [comment]
Oh, I wasn't saying that it _is_ a proper noun. It could be a proper noun in some cases and not in other cases. We have to make the call.

File: WHO_IS_USING_USWDS.md (hunk 1 above)
Comment (maya) [reply WHO_IS_USING_USWDS.md]: Fixed in 2f2aee4.

File: WHO_IS_USING_USWDS.md (hunk 2 above)
Comment (maya) [reply WHO_IS_USING_USWDS.md]: Fixed in 2f2aee4.

**Comment by maya** [comment]
@thisisdano is this also good for approval?

**Pull request merged**